pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -389,43 +521,6 @@ def expected_purchases(
             "chain", "draw", "customer_id", missing_dims="ignore"
         )
 
-    def expected_purchases_new_customer(
```

Comment:
It's worth noting `*new_customer*` methods could support user-specified covariates for what-if and sensitivity analysis, but it's a very niche use case.